### ISSUE TITLE
Player Grappling Balance

### ISSUE THREAD

**Issue by death506** [8thaxisgaming/8thAxis-Public#4]
Player grappling flings grappled players way too far

AOT 1.20.2

In PVP, while grappling onto another player to pull them, it often flings the enemy way too far.  It's also spamable, which gets annoying.
I suggest adding a short cooldown to it and making players be flung less distance when they're grappled by someone else.

**Comment by LilWoetroe** [comment]
Yes this is a bug I need to look into. The current vector calculations are a bit broken and the grappling place location is weirdly placed. For now as you know you can use the world guard flag player-grappling to dissable the feature.

**Comment by LilWoetroe** [comment]
Closing this without a fix since its not a feature thats really suported in the future as I dont believe many players who tried it liked it. In general extremly low priority
